osu.Game.Rulesets.Osu/Edit/Blueprints/Sliders/SliderSelectionBlueprint.cs

```diff
@@ -86,6 +87,10 @@ protected override void LoadComplete()
             pathVersion.BindValueChanged(_ => editorBeatmap?.Update(HitObject));
 
             BodyPiece.UpdateFrom(HitObject);
+
+            if (editorBeatmap != null)
+                selectedObjects.BindTo(editorBeatmap.SelectedHitObjects);
+            selectedObjects.BindCollectionChanged((_, _) => updateVisualDefinition());
```

Comment:
```suggestion
            selectedObjects.BindCollectionChanged((_, _) => updateVisualDefinition(), true);
```

fixes the remaining test failure. As to why, it's a bit of a long story - the long and short of it is that if a selection occurs before `selectedObjects` is bound to `EditorBeatmap`, which happens in the failing test, then the blueprint won't be aware that it has to update its state and will not show the visualiser even if a single object is actually selected.